### ISSUE TITLE
Navi CSS over-riding contentful CSS pages

### ISSUE THREAD

**Issue by Oanh-and-only-Oanh** [topcoder-platform/community-app#3532]
**Describe the bug**
scroll down on any page built on /community
the nav CSS is over riding the CSS for it'

https://www.topcoder.com/community/learn

**Screenshots**
![image](https://user-images.githubusercontent.com/47536078/71196298-a20b6e00-2244-11ea-9b1c-8de2905a1106.png)

**Comment by luizrrodrigues** [comment]
@Oanh-and-only-Oanh  I see the issue.

----

Need check if new nav apply background to `body`

![image](https://user-images.githubusercontent.com/7029641/71196978-5fab5680-226f-11ea-8243-5021463d04a1.png)

**Comment by kkartunov** [comment]
@luizrrodrigues yes, it does. I alerted you many times about it https://github.com/topcoder-platform/navigation-component/blob/5e91a32da09288cf18f79e18f10d7efe4a364ffd/src/assets/sass/style.scss#L10

Also, have this https://github.com/topcoder-platform/navigation-component/issues/68

**Comment by luizrrodrigues** [comment]
@kkartunov My comment above was 4 days ago, before you post https://github.com/topcoder-platform/navigation-component/issues/68, also 68 was marked as duplication because this one already open.
